workflows/DATA_TO_MODEL.nf

```diff
@@ -5,6 +5,8 @@ nextflow.enable.dsl = 2
 params.submissions = ""
 // The expected file type for the submissions
 params.file_type = "csv"
+// The challenge task for which the submissions are made
+params.challenge_task = "task1"
```

Comment:
I was thinking on this yesterday... would it be simpler to keep the type as an int? e.g. `1` for task 1, `2` for task 2, etc.

Or should we keep it a string, should organizers decide use more explicit task names, e.g. "classification_task", "binary_task", etc? 

idk what the best move would be.

Reply:
> Or should we keep it a string, should organizers decide use more explicit task names, e.g. "classification_task", "binary_task", etc?

With this I wonder:
1. How many task variations are there?
2. Is it possible to have multiple of the same type of task in a single challenge? We would still have to use enumeration if so.

As for keeping it simple with `1` instead of `task1` etc, I like that idea, and it feels more straightforward. I'm fine with doing it that way, and changing the parameter from `challenge_task` to `task_number`. Maybe we could do some handling on the template side to always ensure that the input is the datatype we expect (i.e. if the challenge organizer feeds a string `"1"` we force convert it to an `int` before routing).

Reply:
> Is it possible to have multiple of the same type of task in a single challenge?

Yes, very possible! For example, BraTS has 5 segmentation tasks, 2 image synthesis tasks, and another 2 classification tasks.

> As for keeping it simple with 1 instead of task1 etc, I like that idea

Yeah, I'm the type who likes the shorthand version better than the longer explicit way... so I think as a user, I'd much prefer just writing `"1"` than `"task1"` 😅

Reply:
OK sounds good! In that case, for now we will stick with enumerating the tasks instead of naming them, and the enumeration will be `1`, `2`, etc instead of `task1`, `task2` etc.

In the next commit I'll change `challenge_task` to `task_number` so it's clear what kind of input we expect.